externs/ngeox.js

```diff
@@ -258,6 +258,43 @@ ngeox.format.FeatureHashOptions.prototype.properties;
  */
 ngeox.interaction;
 
+
+/**
+ * @typedef {{
+ *     pixelTolerance: (number|undefined),
+ *     style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *     features: ol.Collection.<ol.Feature>,
+ *     wrapX: (boolean|undefined)}}
+ * @api
+ */
+ngeox.interaction.ModifyOptions;
```

Comment:
Yes we were. We're currently in a comment thread in an outdated commit of the diff, if you go to https://github.com/camptocamp/ngeo/pull/986/files it'll be up-to-date.

I'll squash the commits of the PR into a single one once you say this is ready to merge.